build.rs

```diff
@@ -71,6 +71,9 @@ mod download {
     }
 
     pub(crate) fn start() -> anyhow::Result<()> {
+        if std::env::var_os("SKIP_DOWNLOAD").is_some() {
```

Comment:
It's ok if we don't care about the `SKIP_DOWNLOAD` value? ie if `SKIP_DOWNLOAD` is set to `"false"` or `0` and we skip the download

Reply:
That's what I intended, caller can unset the var if it doesn't need it.
I can change it as preferred of course